parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -126,22 +122,18 @@ wait_for_file 3600 1 {{GetCustomSearchDomainsCSEScriptFilepath}} || exit $ERR_FI
 {{GetCustomSearchDomainsCSEScriptFilepath}} > /opt/azure/containers/setup-custom-search-domain.log 2>&1 || exit $ERR_CUSTOM_SEARCH_DOMAINS_FAIL
 {{end}}
 
-{{- if IsDockerContainerRuntime}}
-ensureDocker
-{{else if IsKataContainerRuntime}}
+{{- if IsKataContainerRuntime}}
```

Comment:
can you do us a favor and remove KataContainer support? I don't think we will ever need that

Reply:
ok will remove all kata runtime references